_posts/2020-05-25-flink-on-zeppelin.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Flink on Zeppelin Notebooks for Interactive Data Analysis"
+date:   2020-05-25T08:00:00.000Z
+categories: news
+authors:
+- zjffdu:
+  name: "Jeff Zhang"
+  twitter: "zjffdu"
+---
+
+The latest release of Apache Zeppelin comes with a redesigned interpreter for Apache Flink (version Flink 1.10+ is only supported moving forward) 
+that allows developers and data engineers to use Flink directly on Zeppelin notebooks for interactive data analysis. In this post, we explain how the Flink interpreter in Zeppelin works, 
+and provide a tutorial for running Streaming ETL with Flink on Zeppelin.
+
+# The Flink Interpreter in Zeppelin 0.9
+
+The Flink interpreter can be accessed and configured from Zeppelin’s interpreter settings page. 
+The interpreter has been refactored so that Flink users can now take advantage of Zeppelin to write Flink applications in three languages, 
+namely Scala, Python (PyFlink) and SQL (for both batch & streaming executions). 
+Zeppelin 0.9 now comes with the Flink interpreter group, consisting of the below five interpreters: 
+
+* %flink     - Provides a Scala environment
+* %flink.pyflink   - Provides a python environment
+* %flink.ipyflink   - Provides an ipython environment
+* %flink.bsql     - Provides a stream sql environment
+* %flink.ssql     - Provides a batch sql environment
+
+Not only has the interpreter been extended to support writing Flink applications in three languages, but it has also extended the available execution modes for Flink that now include:
+* Running Flink in Local Mode
+* Running Flink in Remote Mode
+* Running Flink in Yarn Mode
+
+
+You can find more information about how to get started with Zeppelin and all the execution modes for Flink applications in Zeppelin notebooks in this post. 
+
+
+# Flink on Zeppelin for Stream processing
+
+Performing stream processing jobs with Apache Flink on Zeppelin allows you to run most major streaming cases, 
+such as streaming ETL and real time data analytics, with the use of Flink SQL and specific UDFs. 
+Below we showcase how you can execute streaming ETL using Flink on Zeppelin: 
+
+You can use Flink SQL to perform streaming ETL by following the steps below 
+(for the full tutorial, please refer to the Flink Tutorial/Streaming ETL tutorial of the Zeppelin distribution):
```

Comment:
I think there is a link missing here. 

Maybe you can add the link to the Flink Tutorial/Streaming ETL page so that users can go directly there from the post?